AutomatedTesting/Gem/PythonTests/EditorPythonTestTools/editor_python_test_tools/editor_component/editor_component_validation.py

```diff
@@ -12,7 +12,9 @@
 from editor_python_test_tools.editor_entity_utils import EditorComponent
 from consts.general import ComponentPropertyVisibilityStates as PropertyVisibility
 from editor_python_test_tools.asset_utils import Asset
-
+from editor_python_test_tools.editor_component.editor_script_canvas import (ScriptCanvasComponent,
```

Comment:
ScriptCanvasComponent & VariableState appear to be unused.

Reply:
VariableState is now used by the validation file.